src/Sarif.Viewer.VisualStudio/Models/StackFrameModel.cs

```diff
@@ -44,8 +44,8 @@ public int Line
                 if (value != this._line)
                 {
                     this._line = value;
-                    NotifyPropertyChanged("Line");
-                    NotifyPropertyChanged("FullyQualifiedLocation");
+                    NotifyPropertyChanged();
```

Comment:
Why twice?

Reply:
Because a change to this property also affects the value returned from the "full qualified location" property which is computed in the "getter" below.

---
In reply to: [481408972](https://github.com/microsoft/sarif-visualstudio-extension/pull/191#discussion_r481408972) [](ancestors = 481408972)